.github/actions/terraform_state_artifact/action.yml

```diff
@@ -0,0 +1,210 @@
+        }
+
+        # workflows
+        $workflows = twilio api:taskrouter:v1:workspaces:workflows:list --workspace-sid $tf_workspace_sid -o json | ConvertFrom-Json
+
+        $tf_workflow_sid_assign_to_anyone = $workflows | where {$_.friendlyName -eq 'Assign to Anyone'} | Select -property sid -ExpandProperty sid
+        $tf_workflow_sid_chat_transfer = $workflows | where {$_.friendlyName -eq 'Chat Transfer'} | Select -property sid -ExpandProperty sid
+        $tf_workflow_sid_callback = $workflows | where {$_.friendlyName -eq 'Callback'} | Select -property sid -ExpandProperty sid
+        $tf_workflow_sid_internal_call = $workflows | where {$_.friendlyName -eq 'Internal Call'} | Select -property sid -ExpandProperty sid
+
+
+        if ($tf_workflow_sid_assign_to_anyone) {
+          terraform import module.taskrouter.twilio_taskrouter_workspaces_workflows_v1.assign_to_anyone $tf_workspace_sid/$tf_workflow_sid_assign_to_anyone
+        }
+
+        if ($tf_workflow_sid_chat_transfer) {
+          terraform import module.taskrouter.twilio_taskrouter_workspaces_workflows_v1.chat_transfer $tf_workspace_sid/$tf_workflow_sid_chat_transfer
+        }
+
+        if ($tf_workflow_sid_callback) {
+          terraform import module.taskrouter.twilio_taskrouter_workspaces_workflows_v1.callback $tf_workspace_sid/$tf_workflow_sid_callback
+        }
+
+        if ($tf_workflow_sid_internal_call) {
+          terraform import module.taskrouter.twilio_taskrouter_workspaces_workflows_v1.internal_call $tf_workspace_sid/$tf_workflow_sid_internal_call
+        }
+
+        # queues
+        $queues = twilio api:taskrouter:v1:workspaces:task-queues:list --workspace-sid $tf_workspace_sid -o json | ConvertFrom-Json
+
+        $tf_queue_sid_everyone = $queues | where {$_.friendlyName -eq 'Everyone'} | Select -property sid -ExpandProperty sid
+        $tf_queue_sid_template_example_sales = $queues | where {$_.friendlyName -eq 'Template Example Sales'} | Select -property sid -ExpandProperty sid
+        $tf_queue_sid_template_example_support = $queues | where {$_.friendlyName -eq 'Template Example Support'} | Select -property sid -ExpandProperty sid
+        $tf_queue_sid_internal_calls = $queues | where {$_.friendlyName -eq 'Internal Calls'} | Select -property sid -ExpandProperty sid
+
+        if ($tf_queue_sid_everyone) {
+          terraform import module.taskrouter.twilio_taskrouter_workspaces_task_queues_v1.everyone $tf_workspace_sid/$tf_queue_sid_everyone
+        }
+
+        if ($tf_queue_sid_template_example_sales) {
+          terraform import module.taskrouter.twilio_taskrouter_workspaces_task_queues_v1.sales $tf_workspace_sid/$tf_queue_sid_template_example_sales
+        }
+
+        if ($tf_queue_sid_template_example_support) {
+          terraform import module.taskrouter.twilio_taskrouter_workspaces_task_queues_v1.support $tf_workspace_sid/$tf_queue_sid_template_example_support
+        }
```

Comment:
These need to be updated to `template_example_sales` and `template_example_support` instead of just `sales` and `support`.